virtualization/windowscontainers/deployment/docker_windows.md

```diff
@@ -169,7 +169,7 @@ Now, on your machine, open a PowerShell or CMD session, and run the Docker comma
 .\docker.exe -H tcp://<ip address of Nano Server>:2375
 ```
 
-For example, if you would like to see the available images: 
+For example, if you would like to see the available images:
 
 ```powershell
 .\docker.exe -H tcp://<ip address of Nano Server>:2375 images
```

Comment:
I think here a remote setup is wanted as the client runs on your machine and the Nano Server is another machine with different IP address.